### DISCUSSION TITLE
自己按照官方步骤编译出来的be体积很大，但是官方提供的安装包里be却小得多

### DISCUSSION THREAD

**Discussion by xiongzegang** [apache/doris#13371] [A - General / Q&A] [answered]
我自己用最新的代码编译出来的be/lib/doris_be大小为2.1G，官方提供的安装包里该文件只有603M？请问怎么缩小自己编译出来该文件的体积？

**Comment by yuanyuan8983** [comment, marked as answer]
你好，自己执行编译时默认会把DEBUG_INFO编译进去，所以编译出来的doris_be会比较大些，但是带debug的版本定位问题比较方便，如果想小点的话，在编译时执行：STRIP_DEBUG_INFO=ON  sh build.sh 就不会将DEBUG_INFO编译进去了，文件也会相对小点

**Reply by xiongzegang** [reply]
非常感谢
